src/errors.rs

```diff
@@ -125,7 +125,7 @@ impl std::fmt::Display for OciError {
 /// A struct that holds a series of OCI errors
 #[derive(serde::Deserialize, Debug)]
 pub struct OciEnvelope {
-    pub(crate) errors: Vec<OciError>,
+    pub errors: Vec<OciError>,
```

Comment:
Can you add some inline documentation please? This is required by our test suite

Reply:
Sorry, I forgot that. Is this OK?

```rust
/// A struct that holds a series of OCI errors
#[derive(serde::Deserialize, Debug)]
pub struct OciEnvelope {
    /// List of OCI registry errors
    pub errors: Vec<OciError>,
}
```

Reply:
Sure! :heart: